.github/workflows/phpstan.yml

```diff
@@ -56,4 +52,4 @@ jobs:
         with:
           configuration: "phpstan.neon"
           php_version: "${{ matrix.php }}"
-          memory_limit: 16M
+          # memory_limit: "16M"
```

Comment:
`memory_limit: 256M` :)

Reply:
IIRC i tried bumping it up, but higher values did not work (like were rejected).